### ISSUE TITLE
Question on train data sample

### ISSUE THREAD

**Issue by jackie930** [saicoco/Gluon-PSENet#3]
hi,

I am wondering can you provide a train data sample format?
If I want to use my own dataset to fine-tune, I am wondering, when I label the data, should I label the text information on it or simply the BBOX and a class 'text' is fine. 

Thanks!

**Comment by saicoco** [comment]
Sorry to repley to you so late, the training data format is same as ICDAR2015, no need to label text information, just label a text with bbox😊
